sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -212,13 +212,23 @@ opentitan_functest(
             "--rom-kind=rom",
             "--bitstream=\"$(location //hw/bitstream:rom)\"",
             "--bootstrap=\"$(location {flash})\"",
+            "--otp-unprogrammed",
```

Comment:
We should iterate over different OTP configurations, right?

Reply:
SG, thanks!

Reply:
LC state iteration is added in #16376. However we still don't test AST init configs, so the "--otp-unprogrammed" flag is still set.